src/test/recovery/t/039_end_of_wal.pl

```diff
@@ -448,7 +448,7 @@ sub advance_to_record_splitting_zone
 	build_record_header(2 * 1024 * 1024 * 1024, 0, 0xdeadbeef));
 $log_size = -s $node->logfile;
 $node->start;
-ok($node->log_contains("invalid magic number 0000 ", $log_size),
+ok($node->log_contains("invalid magic number BEEF ", $log_size),
 	"xlp_magic zero (split record header)");
```

Comment:
I think it's also worth updating the comment. I also think it's worth adding to the description of the PR about the number of bytes at the end of the page. Something like: in GPDB, 8 bytes remain at the end of the page instead of 16 as in Postgres, so at the beginning of the next page there is a part of the xl_prev field instead of zeros.

Reply:
added

Reply:
I mean: "xlp_magic zero (split record header)"

Reply:
updated